src/Appwrite/Utopia/Response/Model/HealthCertificate.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class HealthCertificate extends Model
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('name', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Name of the service.',
+                'default' => '',
+                'example' => 'database',
+            ])
+            ->addRule('payload', [
+                'type' => self::TYPE_JSON,
+                'description' => 'Certificate information payload',
+                'default' => [],
+                'example' => [
+                    'name' => '/CN=www.google.com',
+                    'validFrom' => '1704200998',
+                    'validTo' => '1711458597',
+                    'signatureTypeSN' => 'RSA-SHA256',
+                ],
+            ])
+            ->addRule('status', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Service status. Possible values can are: `pass`, `fail`',
+                'default' => '',
+                'example' => 'pass',
+            ])
+        ;
```

Comment:
Why do we need a status? The HTTP status code should reflect the status of the certificate. 200 for success and a relevant code for the failed certificates